### PR TITLE
Update documentation following CentOS 7 EOL

### PR DESCRIPTION
**Pull request by stormi**
Plus some improvements in the Additional Packages page.

> Before submitting the pull request, you must agree with the following statements by checking both boxes with a 'x'.
> * [x] "I accept that my contribution is placed under the CC BY-SA 2.0 license [1]."
> * [x] "My contribution complies with the Developer Certificate of Origin [2]." 
>
> [1] https://creativecommons.org/licenses/by-sa/2.0/
> [2] https://docs.xcp-ng.org/project/contributing/#developer-certificate-of-origin-dco

### CODE REVIEW

File: docs/management/additional-packages.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ sidebar_position: 4
 
 # Additional packages
 
-The controller domain (dom0) is a privileged Linux VM, based on CentOS.
+The controller domain (dom0) is a privileged Linux VM, part of which is compatible with RHEL/CentOS packages.
```
Comment (olivierlambert) [inline docs/management/additional-packages.md:7]: I like this wording 👍

File: docs/management/additional-packages.md [hunk 2]
```diff
@@ -118,6 +122,10 @@ yum install name_of_package --enablerepo=epel,base,updates
 
 And as above make sure no package from the system will get overwritten in the process.
 
+:::warning
+EPEL 7 reached its end of life at the same time as CentOS 7, so installing additional packages from its repositories is even more at your own risk now. Let us know what packages from there you really can't use XCP-ng without, so that we may consider providing them in our supported repositories.
```
Comment (olivierlambert) [inline docs/management/additional-packages.md:126]: Maybe adding that supporting those packages mean backporting security fixes when possible?

**Review by olivierlambert — CHANGES_REQUESTED** [review]

File: docs/management/additional-packages.md (hunk 2 above)
Comment (stormi) [reply docs/management/additional-packages.md:126]: Done

**Review by olivierlambert — APPROVED** [review]

File: docs/project/development-process/what-xcp-ng-is-made-of.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ What's inside XCP-ng?
...
+* **CentOS**: many RPM packages come from the [CentOS](https://www.centos.org/) Linux distribution, which in turn is based on Red Hat's [RHEL](https://en.wikipedia.org/wiki/Red_Hat_Enterprise_Linux), mostly based itself on the work of the [Fedora](https://getfedora.org/) project, all based on the work of all the developers who wrote the [FLOSS](https://en.wikipedia.org/wiki/Free/Libre_Open_Source_Software) software that is packaged in those Linux distributions. Examples: glibc, GNU coreutils, crontabs, iptables, many more. However, the list keeps shrinking while we rebuild, patch or update sensitive components directly in XCP-ng repositories.
```
[the remark this answers is not in the document]
Comment (stormi) [reply docs/project/development-process/what-xcp-ng-is-made-of.md:8]: I even replaced it by "and many others"

File: docs/management/additional-packages.md [hunk 4]
```diff
@@ -4,9 +4,9 @@ sidebar_position: 4
 
 # Additional packages
 
-The controller domain (dom0) is a privileged Linux VM, based on CentOS.
+The controller domain (dom0) is a privileged Linux VM, part of which is compatible with RHEL/CentOS packages.
 
-It may be useful to add more packages to it, with precaution. The XCP-ng project offers some in its repositories, and other packages can be installed from CentOS, EPEL, or even third party repositories.
+It may be useful to add more packages to it, with precaution. The XCP-ng project offers some in its repositories, and other packages can be installed from CentOS, EPEL, or even third party repositories, but as the user's own risks.
```
[the remark this answers is not in the document]
Comment (stormi) [reply docs/management/additional-packages.md:9]: Done

**Review by bleader — APPROVED** [review]

**Pull request merged**